ember-browser-services/src/test-support/window-mock-augments.ts

```diff
@@ -1,56 +0,0 @@
-import window from 'ember-window-mock';
-
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-expect-error
-import locationFactory from 'ember-window-mock/test-support/-private/mock/location';
-
-const AUGMENTS: Array<string | symbol> = ['origin'];
```

Comment:
see the [changelog comment](https://github.com/CrowdStrike/ember-browser-services/pull/399/files#diff-b9cc623d4d04c076a2cadea0ace8062484e9e0896e98f6de7f8e79a04ac10b0cR7)